addons/payment/views/payment_templates.xml

```diff
@@ -98,7 +89,7 @@
                         <!-- === Inline form content (filled by acquirer) === -->
                         <t t-set="inline_form_xml_id"
                            t-value="acquirer.sudo().inline_form_view_id.xml_id"/>
-                        <t t-if="inline_form_xml_id">
+                        <t >
                             <div class="clearfix">
```

Comment:
You didn't move the t-if :eyes:, it is on purpose?

Reply:
nope